extensions/wikia/PaidAssetDrop/js/PaidAssetDrop.desktop.js

```diff
@@ -0,0 +1,138 @@
+define('ext.wikia.PaidAssetDrop', [
+	'jquery',
+	'wikia.log',
+	'wikia.window'
+], function ($, log, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.PaidAssetDrop',
+		articleContentId = '#mw-content-text',
+		assetArticleName = 'MediaWiki:PAD_desktop.html',
```

Comment:
I'm fine with hiding the Paid Asset Drop behind abbreviation for the article name, so it's more cryptic for the wiki users.